src/Integration.UnitTests/Binding/BindingControllerTests.cs

```diff
@@ -82,8 +82,7 @@ public void TestInitialize()
 
             folderWorkspaceServiceMock = new Mock<IFolderWorkspaceService>();
             var mefHost = ConfigurableComponentModel.CreateWithExports(
-                MefTestHelpers.CreateExport<IFolderWorkspaceService>(folderWorkspaceServiceMock.Object),
-                MefTestHelpers.CreateExport<IProjectToLanguageMapper>(Mock.Of<IProjectToLanguageMapper>()));
```

Comment:
Miscellaneous: I noticed the `IProjectToLanguageMapper` wasn't required to I removed it.